modules/apps/notification/notification-service/src/main/java/com/liferay/notification/service/impl/NotificationQueueEntryLocalServiceImpl.java

```diff
@@ -184,6 +185,16 @@ public NotificationQueueEntry resendNotificationQueueEntry(
 		NotificationQueueEntry notificationQueueEntry =
 			getNotificationQueueEntry(notificationQueueEntryId);
 
+		if (notificationQueueEntry.getStatus() ==
+				NotificationQueueEntryConstants.STATUS_SENT) {
+
+			throw new NotificationQueueEntryStatusException(
+				String.format(
+					"Must not resend notification with Id %d, because it has " +
```

Comment:
Did you find examples of exception similar to this one? 
I found examples using the term ID. Could you change the word "Id" to "ID", please?

Reply:
You're right! Yes, I just did it